tools/preprocess.py

```diff
@@ -453,20 +466,60 @@ def extract_file(filepath):
     return filepath.parent
 
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description='Preprocess Datasets')
-    parser.add_argument('dataset', metavar='dataset', type=str, help='Dataset to preprocess')
-    parser.add_argument('output_directory', metavar='output_directory', type=str, help='Directory to put graph data')
-    parser.add_argument('--num_partitions', metavar='num_partitions', required=False, type=int, default=1,
+def setArgs():
+    parser = argparse.ArgumentParser(
```

Comment:
Usage is still unclear, we should improve this according to our conversation over slack